frontend/src/components/icons/IconArrowBack.vue

```diff
@@ -0,0 +1,8 @@
+<template>
```

Comment:
tahle zmena je kvuli to mu ze ikony byly mensi nez ve Figma https://www.figma.com/file/9BY7pSQUm6OLlX1zsY1u1i/VK-Design-system?node-id=180%3A2003